src/test/scala/uk/gov/hmrc/test/ui/pages/section6/ContainerListPage.scala

```diff
@@ -20,11 +20,13 @@ import uk.gov.hmrc.test.ui.pages.base.BasePage
 import uk.gov.hmrc.test.ui.pages.base.Constants.{yes, yesNo}
 import uk.gov.hmrc.test.ui.pages.section1.DeclarationChoicePage.isSupplementary
 import uk.gov.hmrc.test.ui.pages.section6.DetailKeys.{ContainerLabel, ExpressConsignment, section6}
+import uk.gov.hmrc.test.ui.pages.section6.InlandModeOfTransportPage.{isFixedTransport, isPostalOrMail}
 
 object ContainerListPage extends BasePage {
 
   def backButtonHref: String =
-    if (isSupplementary) ExpressConsignmentPage.backButtonHref
+    if (isSupplementary && (isFixedTransport || isPostalOrMail)) InlandModeOfTransportPage.path
```

Comment:
Compare to how it was before you added this additional condition. Are you sure that it is necessary? That it was not working due to a missing `clear`?